doc/examples/pulsating-sphere-displacement.py

```diff
@@ -0,0 +1,72 @@
+"""
+Particle displacement of a pulsating sphere.
+"""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation, patches
+
+# Pulsating sphere
+x0 = [0, 0, 0]  # position
```

Comment:
What about calling this `center`?
Or `position`?

Reply:
Other example files are using similar names like `x0` or `xs`.
Should we rename those too?

Reply:
I don't know. Probably.

I just find it absurd to call something `x0` and add a comment saying "position".
If `x0` is already communicating the fact that it is a position, you should remove the comment. If not, you should probably rename the variable (and remove the comment as well).

Reply:
I got your point. You're right.